recipe/run_test.sh

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env bash
+
+if [[ "${target_platform}" == linux-* ]]; then
+  # Hack to help the gn build tool find alsa during build. We can't add ${PREFIX}/${BUILD}/sysroot/lib64 to the
+  # LD_LIBRARY_PATH below because it causes segfaults in many system applications.
+  ln -s ../../lib64/libasound.so.2 ${PREFIX}/${BUILD/conda_cos7/conda}/sysroot/usr/lib64/libasound.so.2
+
+  # Add runtime path of libEGL.so.1 so Qt libraries can find it as they're loaded in.
+  # This must be done before the python interpreter starts up.
+  export LD_LIBRARY_PATH="${PREFIX}/${BUILD/conda_cos7/conda}/sysroot/usr/lib64:${LD_LIBRARY_PATH}"
+fi
+
+${PYTHON} ${RECIPE_DIR}/check_imports.py
+
+shiboken6 --help
+pyside6-rcc -help
+pyside6-uic -help
+test -f ${PREFIX}/lib/qt6/plugins/designer/libPySidePlugin${SHLIB_EXT}
```

Comment:
Could you test that these files (or the core part of them) exist, for example, on `linux`:
```
bin/pyside6-rcc
bin/pyside6-uic
bin/shiboken6
include/PySide6/pyside.h
include/shiboken6/shiboken.h
lib/libpyside6.abi3.so
lib/libpyside6.abi3.so.6.7
lib/libpyside6.abi3.so.6.7.2
lib/libpyside6qml.abi3.so
lib/libpyside6qml.abi3.so.6.7
lib/libpyside6qml.abi3.so.6.7.2
lib/libshiboken6.abi3.so
lib/libshiboken6.abi3.so.6.7
lib/libshiboken6.abi3.so.6.7.2
lib/pkgconfig/pyside6.pc
lib/pkgconfig/shiboken6.pc
```

I found them here https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Fpyside6-6.7.2-py39h0383914_4.conda